libs/auth/src/angular/two-factor-auth/two-factor-auth.component.html

```diff
@@ -0,0 +1,108 @@
+<ng-container *ngIf="loading">
+  <div class="tw-flex tw-items-center tw-justify-center">
+    <i class="bwi bwi-spinner bwi-spin bwi-3x" aria-hidden="true"></i>
+  </div>
+</ng-container>
+
+<ng-container *ngIf="!loading">
+  <form
+    [bitSubmit]="submit"
+    [formGroup]="form"
+    autocomplete="off"
+    (keydown.enter)="handleEnterKeyPress()"
+  >
+    <app-two-factor-auth-email
+      [tokenFormControl]="tokenFormControl"
+      *ngIf="selectedProviderType === providerType.Email"
+    />
+
+    <app-two-factor-auth-authenticator
+      [tokenFormControl]="tokenFormControl"
+      *ngIf="selectedProviderType === providerType.Authenticator"
+    />
+    <app-two-factor-auth-yubikey
+      [tokenFormControl]="tokenFormControl"
+      *ngIf="selectedProviderType === providerType.Yubikey"
+    />
+
+    <app-two-factor-auth-duo
+      (tokenEmitter)="submit($event)"
+      [providerData]="selectedProviderData"
+      *ngIf="
+        selectedProviderType === providerType.OrganizationDuo ||
+        selectedProviderType === providerType.Duo
+      "
+      #duoComponent
+    />
+    <bit-form-control *ngIf="!hideRememberMe()">
+      <bit-label>{{ "dontAskAgainOnThisDeviceFor30Days" | i18n }}</bit-label>
+      <input type="checkbox" bitCheckbox formControlName="remember" />
+    </bit-form-control>
+
+    <app-two-factor-auth-webauthn
+      (webAuthnResultEmitter)="submit($event.token, $event.remember)"
+      (webAuthnInNewTabEmitter)="webAuthInNewTab = $event"
+      *ngIf="selectedProviderType === providerType.WebAuthn"
+    />
+
+    <ng-container *ngIf="selectedProviderType == null">
+      <p bitTypography="body1">{{ "noTwoStepProviders" | i18n }}</p>
+      <p bitTypography="body1">{{ "noTwoStepProviders2" | i18n }}</p>
+    </ng-container>
+
+    <!-- Buttons -->
+    <div class="tw-flex tw-flex-col tw-space-y-3 tw-mb-3">
```

Comment:
```suggestion
    <div class="tw-flex tw-flex-col tw-space-y-3">
```

AnonLayout already has padding built-in.

Reply:
f43b33db6e12b5e5ac134fdfe8694eccaaeab269